Projects/UOContent/Engines/Quests/Uzeraan Turmoil/Items/SchmendrickApprenticeCorpse.cs

```diff
@@ -27,6 +27,8 @@ public SchmendrickApprenticeCorpse() : base(GetOwner(), GetHair(), GetFacialHair
 
         _lantern = new Lantern { Movable = false, Protected = true };
         _lantern.Ignite();
+
+        Owner = null;
```

Comment:
Corpse holds onto a reference to the Owner until it decays. We can get rid of it immediately after the constructor runs.